chore: add to-do list prototype

zairahira opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #393

<!-- Feel free to add any additional description of changes below this line -->

jdwilkin4 commented on frontend-cert/css-projects/todo-list-stylized/user-stories.md:
```diff
@@ -0,0 +1,21 @@
+
+5. The list items should have a bottom margin of `15px` and a line height of `1.6`.
+
+6. Each list item should have squared bullets.
```
They haven't learned about list style type here, so we should remove this requirement

zairahira replied:
In the project sheet, there is a lecture with the title "How to work with the different list-style properties" just above this lab.
I assumed that some list style types would be taught there.
Should I still remove the `list-style-type` from the prototype?

moT01 reviewed (commented):
Generally, I like this @zairahira 👍 - but here's a couple notes...

All the links should have `target=_blank` I think.

It's a little strange that edit and delete buttons link to external URLs. Can we find another way to include a link with those user stories? Like maybe make the link part of the todo list item? "All of your todo items should have a link in them" maybe? If we did that, we could remove the edit and delete buttons.

It also feels a little strange to have the checkbox and the squares on the left. I would be fine with that - just mentioning it.

zairahira commented:
@moT01, thanks for the feedback. I have made the following changes:

- Removed `delete` and `edit` buttons.
- Added a sub-item in the list item that should use a link.
- Removed the sqaure bullet for each main item.
- The sub-items should have a `list-style-type` of `square`.
- Links now have `target=_blank`.
- The links' `href` is currently `#`.

If these look fine, I'll update the user stories.

moT01 commented:
Yea, I think that looks good @zairahira 👍 

I would suggest to format your HTML and there's some comments in the CSS you could remove.
